test/integration.test.js

```diff
@@ -50,17 +50,18 @@ test.serial('Verify GitLab auth', async t => {
 test.serial('Publish a release', async t => {
   const owner = 'test_user';
   const repo = 'test_repo';
+  const repoId = `${owner}%2F${repo}`;
   process.env.GL_TOKEN = 'gitlab_token';
   const nextRelease = {gitHead: '123', gitTag: 'v1.0.0', notes: 'Test release note body'};
   const options = {branch: 'master', repositoryUrl: `https://gitlab.com/${owner}/${repo}.git`};
 
   const gitlab = authenticate()
-    .get(`/projects/${owner}%2F${repo}`)
+    .get(`/projects/${repoId}`)
```

Comment:
No need for this change as the `repoId ` is used only in the URI not in the payload.